### DISCUSSION TITLE
How to use command line variable with dbt power user?

### DISCUSSION THREAD

**Discussion by siwangs** [AltimateAI/vscode-dbt-power-user#439] [General]
I have a date variable in my dbt models to pass for compiling and run, how can pass command line variable to be used with dbt power user

**Comment by ghost** [comment]
There is a setting for passing command line params to dbt.
"dbt.buildModelCommandAdditionalParams":

There you can add more parameters.

However I am not able to make it work.

	"dbt.buildModelCommandAdditionalParams": [
		"--profile development",
		"--vars \"{\\\"source_database\\\": \\\"${env:DBT_SOURCE_DATABASE}\\\", \\\"source_schema\\\": \\\"${env:DBT_SOURCE_SCHEMA}\\\"}\" "

	],

I can't override the profile. It must be set in the yaml file.
Also the vars are not presented and the extension crashes because it doesn't see them.

**Reply by rightx2** [reply]
@ibardarov-fms What about when just `executing` query? (play button on toolbar), not `build` or `run`. There is no such an option(such as `executeModelCommandAdditionalParams`) for `executition`

**Reply by ghost** [reply]
I have to test it the play button on toolbar.

The option which I refer is [1] . 
It would be nice if it can override the profile, and start evaluating the environment variables.

[1] https://github.com/AltimateAI/vscode-dbt-power-user/blob/02ecc8d6b0cd4be708bc1a070d9dd5505942c747/package.json#L108

**Comment by BenEaston** [comment]
Any update on how to pass vars from the extension?
